src/collections/blog/2022/2022-06-13-aaditya-gsoc-blogpost/index.mdx

```diff
@@ -0,0 +1,54 @@
+
+As I grew up I was introduced to a broad spectrum of coding languages that enabled us to make the software that we used on the computer. These languages piqued my interest even more, and I started to explore and make projects. The more I understood them the more interesting it got. Slowly I realized that I just don’t want to make projects that satisfy my needs. I want to build something that will be on the internet and will be used by people around the world. With this goal in my mind, I started exploring and hit a gemstone that is Open source. I would be working on real-world programming solutions that will be used by people all around the world. This helped me to get started and got me excited about contributing to open-source projects. 
+
+### First PR at Layer5 
+I still remember the first issue I worked on for Layer5 was the auto labeler on a pull request. I saw a good first issue tag and still not knowing what the task was, I stated my interest to work on the issue. The issue required knowledge of GitHub Actions which at that time I didn’t even know there was such a thing. After getting the issue assigned, I started researching what GitHub Actions were. I found out that these actions helped to automate workflows that help in building, testing, and deploying software. After getting the basics down I looked for if there are any GitHub Actions already present for the auto labeling of the PR, and to my surprise, there were a couple of them. 
+
+So before actually making a PR, I experimented with the Actions extensively in my private repo. I remember making about 20-25 PR in my private repo just to get the auto labeler working fine. After I verified I got the desired results, I made a PR. I also got to discuss my approach in the development call. I was so nervous at that time since that was the first meeting I attended where I had to make people understand what I did and what approaches I took. I was quite thrilled when my code got merged. I was enthralled to see that my code was working and being useful to the community. This motivated me to make more and more contributions. 
+
+<img src={aadityaFirstPr} alt="Aaditya's first PR merged" />
+
+### A Journey Never to forget 
+
+With this small issue, my journey on Layer5 started. I gradually started learning about the project and started making some impactful contributions. I started to engage in the community more prominently. With the help of these contributions, I learned new technologies, and techniques. Contributing to the codebase gave me so much exposure to React.js, that it would not have been possible if I did it on my own. I took up issues that used React.js and started solving them, and in no time I was comfortable with it. I am thankful, that I didn’t wait for me to learn more about react and then start contributing. I have always applied this principle, learning more by doing actual stuff rather than sitting there and getting stuck in the tutorial loop. 
```

Comment:
```suggestion
My Layer5 journey began with this small issue. I gradually started learning more about the projects and made some impactful contributions. I started to engage in the community more prominently. With the help of these contributions, I learned new technologies, and techniques. Contributing to the codebase gave me so much exposure to React.js, which would not have been possible if I had done it on my own. I took up issues that used React.js and started solving them, and in no time, I was comfortable with it. I am thankful that I didn’t wait for me to learn more about react and then start contributing. I have always applied this principle, learning more by doing actual stuff rather than sitting there and getting stuck in the tutorial loop. 
```